helm/inji-verify/install.sh

```diff
@@ -31,13 +31,12 @@ fi
 
 echo "MOSIP_INJIVERIFY_HOST is not present in configmap/global of configserver"
     # Add injiverify host to global
-    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INIJIVERIFY_HOST\"}}"
+    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INJIVERIFY_HOST\"}}"
+    kubectl patch configmap global -n default --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INJIVERIFY_HOST\"}}"
```

Comment:
Why do we need this configuration in global namespace?

Reply:
Better it is updatd in global also. Looks good only